meson.build

```diff
@@ -1,46 +1,49 @@
-project(
+# Project name, programming language and version
+project (
     'com.github.peteruithoven.resizer',
-    ['vala', 'c'],
-    meson_version : '>= 0.47',
-    license: 'GPL-3.0+'
+    'c', 'vala',
+    version: '1.0.1'
```

Comment:
I'd like to limit the amount of places where the version is defined, to ideally nowhere but the tag. Is this really useful?

Reply:
I don't really know but elementary does in their app as well as many other developers so.